react/Rating/Rating.js

```diff
@@ -28,17 +39,39 @@ const getStar = (percent, key, starClassName) => {
   return <StarIcon {...props} />;
 };
 
-const Rating = ({ rating, starClassName, ...restProps }) => {
+const Rating = ({
+  rating,
+  starClassName,
+  showTextRating,
```

Comment:
I feel like this should either be a separate PR, or called out in the PR title. The current title makes it sound like only icons are affected.

Reply:
Added to title